doc/docs/Python_Tutorials/Cylindrical_Coordinates.md

```diff
@@ -618,6 +618,18 @@ As a demonstration, we compute the [extraction efficiency of an LED](https://mee
 |   6.7  |          0.321860         |        0.036       |    101   |
 |   9.5  |          0.324270         |        0.028       |    141   |
 
+The extraction efficiency computed thus far is for *all* angles. To compute the extraction efficiency within an angular cone (i.e., as part of an overall calculation of the [radiation pattern](Near_to_Far_Field_Spectra.md#radiation-pattern-of-an-antenna)), we would need to surround the emitting structure with a closed box of near-field monitors. However, because the LED slab is infinitely extended a non-closed box must be used.  This will introduce [truncation errors](Near_to_Far_Field_Spectra.md#radiation-pattern-of-an-antenna) which are unavoidable.
+
+The procedure for computing the extraction efficiency within an angular cone for a dipole source at $r > 0$ involves five steps:
+
+1. For each simulation in the Fourier-series expansion ($m = 0, 1, ..., M$), compute the far fields $\vec{E}_m$, $\vec{H}_m$ for $N$ points along the circumference of a quarter circle spanning the angular range [0°, 90°] with "infinite" radius (i.e., $R \gg \lambda$) using a near-to-far field transformation.
+2. Obtain the *total* far fields by summing the far fields from (1): $\vec{E}_{tot} = \vec{E}_{m=0} + 2\sum_{m=1}^M \Re\{\vec{E}_m\}$ and $\vec{H}_{tot} = \vec{H}_{m=0} + 2\sum_{m=1}^M \Re\{\vec{H}_m\}$.
```

Comment:
If you have a non-axisymmetric source, then this needs to be a function of φ as well — you need to include exp(imφ) terms in the sum.